anta/tests/security.py

```diff
@@ -354,14 +344,27 @@ def test(self) -> None:
 
 
 class VerifyAPISSLCertificate(AntaTest):
-    """Verifies the eAPI SSL certificate expiry, common subject name, encryption algorithm and key size.
+    """Verifies the eAPI SSL certificate.
+
+    This test performs the following checks for each certificate:
+
+      1. Validates that the certificate is not expired and meets the configured expiry threshold.
+      2. Validates that the certificate's common name matches the expected subject name.
+      3. Ensures the certificate uses the specified encryption algorithm.
+      4. Verifies the certificate's key size aligns with the encryption algorithm's requirements.
```

Comment:
The key size on the device cannot "not align" with the encryption algorithm requirement. We in ANTA are checking that people cannot enters absurd inputs by checking that the key size is possible with the encryption algo. But thats not what the test is checking.

 
```suggestion
      4. Verifies the certificate key matches the expected key size.
```

Reply:
Updated